lib/repo/subversion_repository.rb

```diff
@@ -35,19 +35,23 @@ class SubversionRepository < Repository::AbstractRepository
     # created using SubversionRepository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
-        raise ConfigurationError.new("Required config 'IS_REPOSITORY_ADMIN' not set")
+      unless MarkusConfigurator.markus_config_repository_admin?
+        raise ConfigurationError.new("Init: Required config " \
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.